jetstream_pt/engine.py

```diff
@@ -693,24 +689,12 @@ def create_pytorch_engine(
     checkpoint_format = "safetensors"
     checkpoint_path = paths[0]
 
-  env_data = JetEngineEnvironmentData(
-      tokenizer_path=tokenizer_path,
-      checkpoint_path=checkpoint_path,
-      checkpoint_format=checkpoint_format,
-      model_type="llama-2-" + param_size,
-      batch_size=batch_size,
-      max_decode_length=max_decode_length,
-      max_input_sequence_length=context_length,
-      enable_weight_quantization=quantize_weights,
-      enable_kv_quantization=quantize_kv,
-      cache_sequence_length=max_cache_length,
-      bf16_enable=bf16_enable,
-  )
-  env = JetEngineEnvironment(env_data)
-
   tokenizer = token_utils.load_vocab(tokenizer_path)
   pt_model = None
-  if model_name == "llama":
+
+  if model_name.startswith("llama"):
+    from jetstream_pt.third_party.llama2 import model_exportable, model_args
```

Comment:
Can you do import in the head of this file? Otherwise, you get below error: 
jetstream_pt/engine.py:696:4: C0415: Import outside toplevel (jetstream_pt.third_party.llama2.model_exportable, jetstream_pt.third_party.llama2.model_args) (import-outside-toplevel)

Reply:
done.